k8s/k8s.go

```diff
@@ -1526,6 +1532,74 @@ func (k *k8sOps) GetStatefulSetsUsingStorageClass(scName string) ([]apps_api.Sta
 	return retList, nil
 }
 
+func (k *k8sOps) GetPVCForStatefulSet(name, namespace string) (*v1.PersistentVolumeClaimList, error) {
+	ss, err := k.GetStatefulSet(name, namespace)
+	if err != nil {
+		return nil, err
+	}
+
+	listOptions, err := k.getListOptionsForStatefulSet(ss)
+	if err != nil {
+		return nil, err
+	}
+
+	return k.getPVCWithListOptions(namespace, listOptions)
+}
+
+func (k *k8sOps) ValidatePVCForStatefulSet(name, namespace string) error {
```

Comment:
duplicate code here. just call `k.GetPVCForStatefulSet` and then invoke `k.ValidatePersistentVolumeClaim`

`k.ValidatePersistentVolumeClaim` already has a retry loop for the PVC validation.

Reply:
We need the stateful set here to compare with PVC count with replica sets.

This retry also waits for the count of PVCs to match the replica sets.